app/assets/javascripts/cited_documents.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable camelcase */
+/* global CitedDocuments*/
+
+(function (global) {
+  var CitedDocuments;
+
+  CitedDocuments = {
+    init: function (el) {
+      var $el = $(el);
+      var data = $el.data();
+      console.log(data);
+      $.getJSON(data.path, {
```

Comment:
@aeschylus I'm not sure what's the best way to do this but for each related document_id you can use the show route for it to get the title via JSON like so:

  https://exhibits.stanford.edu/parker-2/catalog/wp146tq7625.json

I don't know how to construct the Blacklight JSON API query that allows you to put in multiple document_id's, because this:

  https://exhibits.stanford.edu/parker-2/catalog.json?f[id][]=wp146tq7625&f[id][]=abcd

doesn't seem to work.